packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -28,5 +28,10 @@
     vertical-align: middle;
   }
 
-
+  .ms-Persona.ms-Persona--xs {
```

Comment:
Yup, noted in my new commit.  It's there to prevent truncation of the Persona name